projects/VideoDDSPublisher/CMakeLists.txt

```diff
@@ -16,11 +16,10 @@ find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets REQUIRED)
 
 find_package(GStreamer REQUIRED glib gobject base app video)
 
-find_package(OpenSplice REQUIRED)
-OpenSplice_IDLCPP_GEN(${CommonDir}/VideoDDS.idl ${CMAKE_CURRENT_BINARY_DIR})
+find_package(CycloneDDS-CXX REQUIRED)
```

Comment:
Does this come with Cmake/Cyclone DDS or is it custom made?

Reply:
That comes with Cyclone DDS. Very conveniently.